client/src/components/rounds/Apply.tsx

```diff
@@ -90,8 +92,11 @@ function Apply() {
     );
   }
 
+  // todo: navigating to early...
```

Comment:
Can you elaborate a bit on this comment/todo?
If I had to address it I wouldn't know what it is referring to. Maybe create an issue and add it to the comment?

Reply:
Yeah it seems to redirect before TX is confirmed.

Reply:
I meant if you can add more context to the comment 😛